spec/mongoid/token/collisions_spec.rb

```diff
@@ -35,7 +35,7 @@
         it "should raise an error after retrying" do
           resolver.stub(:retry_count).and_return(3)
           attempts = 0
-          expect{document.resolve_token_collisions(resolver) { attempts += 1; raise Moped::Errors::OperationFailure.new("","") }}.to raise_error Mongoid::Token::CollisionRetriesExceeded
+          expect{document.resolve_token_collisions(resolver) { attempts += 1; raise Mongo::Error::OperationFailure }}.to raise_error Mongoid::Token::CollisionRetriesExceeded
```

Comment:
Line is too long. [173/80]<br>Do not use semicolons to terminate expressions.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.